lib/optimize/ConcatenatedModule.js

```diff
@@ -829,8 +829,6 @@ class ConcatenatedModule extends Module {
 						topLevelDeclarations.add(decl);
```

Comment:
A few lines above there is a `TODO` comment that can be removed.

And I also think the `if (RESERVED_NAMES.has(decl)) continue;` is incorrect here. What do you think?

Reply:
> And I also think the if (RESERVED_NAMES.has(decl)) continue; is incorrect here. What do you think?

maybe basic idea was that `buildInfo.topLevelDeclarations` are useless in this case since renaming will happen anyway, but yeah this is not correct.